### PR TITLE
Add subheading to elections page

### LINKED ISSUE #7136: Add subheading to EU elections pages
## Description

Update elections page with subheading.

EU_Elections_Hero

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

Add subheading to elections page

## Issue / Bugzilla link

Fix #7136.

### CODE REVIEW

File: media/css/firefox/election.scss [hunk 1]
```diff
@@ -18,6 +18,13 @@ $image-path: '/media/protocol/img';
     padding-bottom: 200px;
 }
 
+.mzp-c-hero-title {
+    @include at2x('/media/img/firefox/election/firefox-logo.png', 100px);
```
Comment (alexgibson) [inline media/css/firefox/election.scss]: Suggest adding this to `/img/logos/firefox/` as a central place, so we don't end up with multiple copies of the same wordmark logo?

File: media/css/firefox/election.scss [hunk 2]
```diff
@@ -18,6 +18,13 @@ $image-path: '/media/protocol/img';
...
+    background-repeat: no-repeat;
+    background-position: top left;
```
Comment (alexgibson) [inline media/css/firefox/election.scss]: Should the default position for this be `top center`, to match the behavior of the default hero component?

image

**Comment by stephaniehobson** [comment]
Issue for the "logo lockup" https://github.com/mozilla/protocol/issues/373

File: media/css/firefox/election.scss (hunk 1 above)
Comment (stephaniehobson) [reply media/css/firefox/election.scss]: The same image only twice the size is used on the WN67 page. The size on 67 seems more likely to be the one used centrally so I moved that one. Which means I removed the `a2x` mixin here and just use the low-res version at half its size.

File: media/css/firefox/election.scss (hunk 2 above)
Comment (stephaniehobson) [reply media/css/firefox/election.scss]: ✅

**Pull request merged**